packages/babel-plugin/src/css-map/index.ts

```diff
@@ -98,15 +98,15 @@ export const visitCssMapPath = (
         const { sheets, classNames } = transformCssItems(css, meta);
         totalSheets.push(...sheets);
 
-        if (classNames.length !== 1) {
+        if (classNames.length > 1) {
           throw buildCodeFrameError(
             createErrorMessage(ErrorMessages.STATIC_VARIANT_OBJECT),
             property,
             meta.parentPath
           );
         }
 
-        return t.objectProperty(property.key, classNames[0]);
+        return t.objectProperty(property.key, classNames[0] || t.stringLiteral(''));
```

Comment:
Got it, so having `'&:hover'` empty threw this?

Reply:
Yeah exactly.

Reply:
> Interesting, do you know why this works? I've seen similar things, just never seen like a TS issue or explanation.

I think it's an edge case with complex nested types (interface + omits + ?). At a glace `unknown` should be fine but in this case it's obviously not.

I'm not sure!